src/Nova/Resource.php

```diff
@@ -4,11 +4,94 @@
 
 namespace Tipoff\Support\Nova;
 
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\Field;
+use Laravel\Nova\Fields\ID;
 use Laravel\Nova\Http\Requests\NovaRequest;
 use Laravel\Nova\Resource as NovaResource;
 
 abstract class Resource extends NovaResource
```

Comment:
Should we call this BaseResource instead of Resource? That would allow a similar naming convention with our BaseModel.

Reply:
Your call.

Reply:
OK, I prefer doing that then. I'll make the commit.